input/script_utils.go

```diff
@@ -1115,7 +1144,7 @@ func receiverHtlcTapScriptTree(senderHtlcKey, receiverHtlcKey,
 	// With the two leaves obtained, we'll now make the tapscript tree,
 	// then obtain the root from that
 	tapscriptTree := txscript.AssembleTaprootScriptTree(
-		successTapLeaf, timeoutTapLeaf,
+		timeoutTapLeaf, successTapLeaf,
```

Comment:
Really just cosmetic, a reviewer pointed out it didn't match the order in the spec so I swapped. In practice both are valid since they'll be sorted before they're hashed.